saleor/discount/migrations/0049_migrate_sales_to_promotions.py

```diff
@@ -1,9 +1,11 @@
 # Generated by Django 3.2.18 on 2023-07-06 09:43
+from django.db import migrations
 from dataclasses import dataclass
-from typing import List
+from typing import Dict, List
 
 import graphene
-from django.db import migrations
+from django.db.models import Exists, OuterRef
+
 
 # The batch of size 100 takes ~1.2 second and consumes ~25MB memory at peak
```

Comment:
Did memory usage changed?

Also just reminder, that we will most likely need to move it to task when going to release.

Reply:
I will measure it again

Reply:
@kadewu there will be separate PR with changes for providing zero-downtime approach

Reply:
time down by 0.3s; memory up by 5MB